test/Engine/FFITarget/ClassWithMigratedMembers.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FFITarget
+{
+    public class ClassWithMigratedMembers
+    {
+
+        public static ClassWithMigratedMembers ByNothing()
+        {
+            return new ClassWithMigratedMembers();
+        }
+
+        //we test by opening a file that references a node with three params
+        public string AMemberWithTwoParams(string a, string b)
+        {
+            return a + b;
+        }
+        public string AMemberWithThreeParams(string a, string b,string c ="abc")
+        {
+            return "migrated";
+        }
+
+        //we test a file that uses the node AMemberWithAChangedName0
+        public string AMemberWithAChangedName2(string a)
+        {
+            return a;
+        }
+      /*  public string AMemberWithAChangedName1(string a)
```

Comment:
Does this have to be commented out in order for the migration to work?

Reply:
no, I just thought it was more explicit to illustrate what we were testing, a bit weird I admit.